src/integrationTest/java/org/opensearch/security/api/AccountRestApiIntegrationTest.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ */
+
+package org.opensearch.security.api;
+
+import org.junit.Test;
+
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.test.framework.TestSecurityConfig;
+import org.opensearch.test.framework.cluster.TestRestClient;
+
+import static org.apache.commons.lang3.RandomStringUtils.randomAlphabetic;
+import static org.opensearch.security.dlic.rest.support.Utils.hash;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class AccountRestApiIntegrationTest extends AbstractApiIntegrationTest {
+
+    private final static String TEST_USER = "test-user";
+
+    private final static String RESERVED_USER = "reserved-user";
+
+    private final static String HIDDEN_USERS = "hidden-user";
+
+    public final static String TEST_USER_PASSWORD = randomAlphabetic(10);
+
+    public final static String TEST_USER_NEW_PASSWORD = randomAlphabetic(10);
+
+    static {
+        testSecurityConfig.user(new TestSecurityConfig.User(TEST_USER).password(TEST_USER_PASSWORD))
+            .user(new TestSecurityConfig.User(RESERVED_USER).reserved(true))
+            .user(new TestSecurityConfig.User(HIDDEN_USERS).hidden(true));
+    }
+
+    private String accountPath() {
+        return super.apiPath("account");
+    }
+
+    @Test
+    public void accountInfo() throws Exception {
+        withUser(NEW_USER, client -> {
+            var response = ok(() -> client.get(accountPath()));
+            final var account = response.bodyAsJsonNode();
+            assertEquals(response.getBody(), NEW_USER, account.get("user_name").asText());
+            assertFalse(response.getBody(), account.get("is_reserved").asBoolean());
+            assertFalse(response.getBody(), account.get("is_hidden").asBoolean());
```

Comment:
Nit; I'd prefer if we only used `assertThat(...)` to keep consistency and verbosity in test output.

Reply:
Sure will change.

Reply:
done